packages/pwa/app/pages/product-list/partials/empty-results.jsx

```diff
@@ -66,36 +67,48 @@ const EmptySearchResults = ({searchQuery, category}) => {
                     <Text fontSize={['md', 'md', 'md', 'md']} fontWeight="400">
                         {intl.formatMessage(
                             {
-                                id: 'product_list_page.no_results_double_check',
+                                id: 'empty_search_results.info.double_check_spelling',
                                 defaultMessage: 'Double-check your spelling and try again or {link}'
                             },
                             {
                                 link: (
                                     <Button variant="link" to={'/'}>
-                                        {intl.formatMessage({
-                                            id: 'product_list_page.no_results.contact_us',
-                                            defaultMessage: 'contact us'
-                                        })}
+                                        {intl.formatMessage(contactUsMessage)}
                                     </Button>
                                 )
                             }
                         )}
                     </Text>
                     <Stack spacing={16} marginTop={32}>
                         <RecommendedProducts
-                            title={<FormattedMessage defaultMessage="Top Sellers" />}
+                            title={
+                                <FormattedMessage
+                                    defaultMessage="Top Sellers"
+                                    id="empty_search_results.title.top_sellers"
```

Comment:
Should we use `empty_search_results.recommended_products.title.top_sellers` for more clarity? 🤔

Reply:
I see what you mean. Yeah, we can add 'recommended_products' to be clearer.